plugins/MumblePlugin.h

```diff
@@ -1515,6 +1535,22 @@ struct MUMBLE_API_STRUCT_NAME {
 																			mumble_channelid_t channelID,
 																			const char **description);
 
+#	if SELECTED_API_VERSION >= MUMBLE_PLUGIN_VERSION_CHECK(1, 3, 0)
+
+	/**
+	 *
+	 * Gets the linked channels of the given channel.
+	 *
+	 * @param callerID The ID of the plugin calling this function
+	 * @param channelID The ID of the channel
+	 * @param[out] linkList The set of channel IDs linked to the channelID
+	 * @returns The error code. If everything went well, STATUS_OK will be returned.
+	 */
+	mumble_error_t(MUMBLE_PLUGIN_CALLING_CONVENTION *getLinkedChannels)(mumble_plugin_id_t callerID,
+																		mumble_channelid_t channelID,
+																		std::set< mumble_channelid_t > *linkedList);
```

Comment:
We can't use `std::set` here - the API has to be pure C. That means that we have to use an array (or pointer, really) along with a size parameter.

Reply:
I want to discuss one thing. If I prefer pointers, we need to call malloc, and someone needs to follow the lifetime of this allocation. I mean, API users need to free this allocation. If it is ok, then I don't worry about memory leaks. The other choice is using the array. We need to fix the size in that case, but I prefer a more flexible size. Which one should we choose?

Reply:
`std::array` can't be used either as that too is not C. We only have the option of allocating a buffer and then passing the pointer and the size of that buffer around